tests/test_grpo_trainer.py

```diff
@@ -1415,6 +1415,8 @@ def test_prepare_input_called_with_correct_data(self):
     )
     @require_vision
     def test_training_vlm(self, model_id):
+        if model_id == "trl-internal-testing/tiny-LlavaNextForConditionalGeneration":
+            pytest.xfail("LlavaNext is currently broken, see https://github.com/huggingface/transformers/issues/42968")
```

Comment:
Now that the dtype is properly set, we get a dtype issue, reported in transformers